pkg/services/authz/rbac/mapper.go

```diff
@@ -238,6 +238,11 @@ func NewMapperRegistry() MapperRegistry {
 				folderSupport: false,
 			},
 		},
+		"provisioning.grafana.app": {
```

Comment:
@IevaVasiljeva I also thought I would need this to pass the access checker for OBO use-cases, please let me know if that is not the case.

Reply:
This is needed for user authorization through MT authZ service. So you're right that it would be needed for OBO user MT flows.